test/EDSC/builder-api-test.cpp

```diff
@@ -474,6 +474,64 @@ TEST_FUNC(select_op) {
   f.erase();
 }
 
+TEST_FUNC(select_op_f32) {
+  using namespace edsc;
+  using namespace edsc::intrinsics;
+  using namespace edsc::op;
+  auto f32Type = FloatType::getF32(&globalContext());
+  auto memrefType = MemRefType::get({-1, -1}, f32Type, {}, 0);
+  auto f = makeFunction("select_op", {}, {memrefType, memrefType});
+
+  OpBuilder builder(f.getBody());
+  ScopedContext scope(builder, f.getLoc());
+  // clang-format off
+  ValueHandle zero = constant_index(0), one = constant_index(1);
+  MemRefView vA(f.getArgument(0)), vB(f.getArgument(1));
+  IndexedValue A(f.getArgument(0)), B(f.getArgument(1));
+  IndexHandle i, j;
+  LoopNestBuilder({&i, &j}, {zero, zero}, {one, one}, {1, 1})([&]{
+
+    edsc::intrinsics::select(B(i, j) == B(i+one, j), *A(zero, zero), *A(i, j));
+    edsc::intrinsics::select(B(i, j) != B(i+one, j), *A(zero, zero), *A(i, j));
+    edsc::intrinsics::select(B(i, j) >= B(i+one, j), *A(zero, zero), *A(i, j));
+    edsc::intrinsics::select(B(i, j) <= B(i+one, j), *A(zero, zero), *A(i, j));
+    edsc::intrinsics::select(B(i, j) < B(i+one, j), *A(zero, zero), *A(i, j));
+    edsc::intrinsics::select(B(i, j) > B(i+one, j), *A(zero, zero), *A(i, j));
+  });
+
+  // CHECK-LABEL: @select_op
+  //      CHECK: affine.for %{{.*}} = 0 to 1 {
+  // CHECK-NEXT:   affine.for %{{.*}} = 0 to 1 {
+  //  CHECK-DAG:     {{.*}} = cmpf "oeq"
```

Comment:
Can we drop the '{{.*}} = ' on all of these? It doesn't look related to the test.

Reply:
Out of curiosity, why would that matter ?

Reply:
It's a testing principle to not have things unrelated to what you are testing as part of the test:
https://github.com/tensorflow/mlir/blob/master/g3doc/TestingGuide.md